spark-indexer/src/test/java/org/uniprot/store/spark/indexer/common/store/DataStoreIndexerFactoryTest.java

```diff
@@ -0,0 +1,39 @@
+package org.uniprot.store.spark.indexer.common.store;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+import org.uniprot.store.spark.indexer.uniparc.UniParcDataStoreIndexer;
+import org.uniprot.store.spark.indexer.uniprot.UniProtKBDataStoreIndexer;
+import org.uniprot.store.spark.indexer.uniref.UniRefDataStoreIndexer;
+
+/**
+ * @author lgonzales
+ * @since 08/05/2020
+ */
+class DataStoreIndexerFactoryTest {
+
+    @Test
+    void createUniParcDataStoreIndexer() {
+        DataStoreIndexerFactory factory = new DataStoreIndexerFactory();
+        DataStoreIndexer indexer = factory.createDataStoreIndexer(DataStore.UNIPARC, null);
+        assertNotNull(indexer);
+        assertTrue(indexer instanceof UniParcDataStoreIndexer);
+    }
+
+    @Test
+    void createUniProtKBDataStoreIndexer() {
+        DataStoreIndexerFactory factory = new DataStoreIndexerFactory();
+        DataStoreIndexer indexer = factory.createDataStoreIndexer(DataStore.UNIPROT, null);
+        assertNotNull(indexer);
+        assertTrue(indexer instanceof UniProtKBDataStoreIndexer);
+    }
+
```

Comment:
Negative test for factory is missing.

Reply:
All DataStore are supported, 
I just added the default because  Sonar Lint requested it...